CI/run_reframe.sh

```diff
@@ -50,11 +50,14 @@ fi
 if [ -z "${EESSI_TESTSUITE_BRANCH}" ]; then
     EESSI_TESTSUITE_BRANCH='v0.4.0'
 fi
-if [ -z "${EESSI_CVMFS_REPO}" ]; then
-    export EESSI_CVMFS_REPO=/cvmfs/software.eessi.io
-fi
-if [ -z "${EESSI_VERSION}" ]; then
-    export EESSI_VERSION=2023.06
+if [ -z "${USE_EESSI_SOFTWARE_STACK}" ] | [ $USE_EESSI_SOFTWARE_STACK == "True" ]; then
```

Comment:
I'm getting:
```
$ EESSI_CI_SYSTEM_NAME=surf_snellius ./run_reframe.sh
Running CI on host int5
/tmp/EESSI-test-suite/CI
./run_reframe.sh: line 53: [: ==: unary operator expected
```
Should be resolved by:
```suggestion
if [ -z "${USE_EESSI_SOFTWARE_STACK}" ] | [ "$USE_EESSI_SOFTWARE_STACK" == "True" ]; then
```